### DISCUSSION TITLE
Simplify vscode-java-test installation instructions in README

### DISCUSSION THREAD

**Discussion by gbroques** [mfussenegger/nvim-jdtls#521] [General]
I recently setup this plugin for debugging JUnit tests with [vscode-java-test](https://github.com/microsoft/vscode-java-test), and ran into an issue following [the instructions in the README](https://github.com/mfussenegger/nvim-jdtls/blob/b041dd4308d8c715187274ece9bc8d501a6f3041/README.md?plain=1#L358-L361). Particularly with the `npm run build-plugin` step since `vscode-java-test` depends on a SNAPSHOT version of JDTLS.

See https://github.com/microsoft/vscode-java-test/issues/1580 and https://github.com/microsoft/vscode-java-test/pull/1581 for details.

I noticed looking at @mfussenegger's configuration, linked to from [the Wiki](https://github.com/mfussenegger/nvim-jdtls/wiki/Sample-Configurations#mfussenegger-configuration), that they also had trouble with this step:
https://github.com/mfussenegger/dotfiles/blob/833d634251ebf3bf7e9899ed06ac710735d392da/vim/.config/nvim/ftplugin/java.lua#L109-L110
> -- npm install broke for me: https://github.com/npm/cli/issues/2508
> -- So gather the required jars manually; this is based on the gulpfile.js in the vscode-java-test repo

See discussion https://github.com/mfussenegger/nvim-jdtls/discussions/329 for other problems with `vscode-java-test` bundles.

Building the bundles for `vscode-java-test` may require a certain version of Java, `npm` / Node.js, and fail for an unknown reason.

A better way could be to download the pre-built bundles from the [latest release](https://github.com/microsoft/vscode-java-test/releases).

For example, we can download the [vscjava.vscode-java-test-0.39.0.vsix file](https://github.com/microsoft/vscode-java-test/releases/download/0.39.0/vscjava.vscode-java-test-0.39.0.vsix), extract it, and specify the path to `/extension/server/`.

Does anyone have any thoughts on whether this is a better process than what's currently documented in the README?

**Comment by gbroques** [comment]
@mfussenegger Any thoughts on whether a pull-request would be welcomed for this?

Thank you for the time, and apologies for the mention if you're busy!

**Reply by mfussenegger** [reply]
> Does anyone have any thoughts on whether this is a better process than what's currently documented in the README?

In general by using released artifacts you're less likely to run into temporary breakages, so yes, documenting that approach sounds good to me.

**Reply by gbroques** [reply]
Thanks @mfussenegger! I opened https://github.com/mfussenegger/nvim-jdtls/issues/596 and will work on submitting a PR for this.
